models/demos/deepseek_v3/reference/modeling_deepseek.py

```diff
@@ -407,6 +407,31 @@ def forward(self, hidden_states):
         return topk_idx, topk_weight
 
 
+class DeepseekV3MoE_Experts(nn.Module):
```

Comment:
let's maybe keep this the same as the HF version, and just move this class or forward function to the test itself